demos/demo-minimal-js/index.js

```diff
@@ -115,7 +125,9 @@ export function answerCall() {
 }
 
 export function endCall() {
-  cti.callEnded();
+  cti.callEnded({
+    callEndStatus: "COMPLETED",
```

Comment:
What do you think about our SDK proving call status constants?

Reply:
Should we move demos/demo-react-ts/src/types/EngagementStatuses.ts to the SDK? We would have to publish an alpha version to access the constants but I don't see them changing very much over time.

Reply:
Yeah, I think we should have a PR that merges into `master` deployed as a refactor/patch.